### PR TITLE
Feature/gift entry  ken populate lookup sibling fields

### PR DESCRIPTION
**Pull request by ptewson-sfdo**
# Critical Changes

# Changes

# Issues Closed

# Community Ideas Delivered

# New Metadata

# Deleted Metadata

### CODE REVIEW

File: src/lwc/geFormField/geFormField.js [hunk 1]
```diff
@@ -304,38 +300,48 @@ export default class GeFormField extends LightningElement {
...
 
-    /**
-     * Load a value into the form field.
-     * @param data  An sObject potentially containing a value to load.
-     */
-    @api
-    load(data) {
-        const value = data[this.sourceFieldAPIName];
-        this.setValue(value);
+        if (value === null) {
```
Comment (voduyemi) [inline src/lwc/geFormField/geFormField.js]: @ptewson-sfdo This change actually fixes the first bug in the WI I was working on, so I'll scale back my changes to just cover the look-up issue only.

**Comment by ptewson-sfdo** [comment]
**lurch: attach W-039031

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x1E000001HrMQQA0" target="_blank">W-039031</a>

**Comment by ptewson-sfdo** [comment]
@kenlewis-sfdo I think we should close this PR, or else re-base it to the feature/gift-entry-fast-follow branch (and rename accordingly). What do you think?

**Comment by kenlewis-sfdo** [comment]
Closing due to replacement PR created, parented to the fast follow branch here: https://github.com/SalesforceFoundation/NPSP/pull/5151/files

**Pull request closed without merging**